components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -221,4 +258,119 @@ Should you have a need to revert back to IAM v1, it is quick and easy to do:
 $ chef-automate iam reset-to-v1
 ```
 
-Note that this discards your IAM v2 policies and roles, and re-configures Chef Automate to use your v1 policies again.
+Note that this discards your IAM v2 policies, roles, and projects, and re-configures Chef Automate to use your v1 policies again.
+
+<!-- TODO move this next to the rest of the projects walkthrough -->
+## Assigning Projects to Ingested Resources
+
+While projects can be directly assigned to resources created within Automate, ingested resources, including Compliance reports, events, and nodes, are created outside of Automate and therefore must be tagged with projects a different way.
+
+To tag any ingested resource with a project, that project must have at least one rule. A project rule contains conditions that
+determine if an ingested resource should be tagged with the rule's associated project. Each condition contains an attribute, operator, and value. See the [API reference]({{< relref "iam-v2-api-reference.md#project-rules" >}}) for details on how to manage project rules.
+Take this rule for the project `project-devops` as an example:
+
+```json
+{
+  "id": "devops-rule",
+  "name": "devops rule",
+  "type": "NODE",
+  "project_id": "project-devops",
+  "conditions": [
+    {
+      "operator": "EQUALS",
+      "attribute": "CHEF_ORGANIZATION",
+      "values": [
+        "devops"
+      ]
+    }
+  ]
+}
+```
+
+Once this project rule has been created, it must be applied in order to take effect. Once the project rule is in effect, all ingested resources with a `CHEF_ORGANIZATION` field that has the value `devops` will be considered a part of the `project-devops` project.
+Only these resources will appear in Automate when the `project-devops` project has been selected in the global project filter.
+
+{{% warning %}}
+Compliance reports must be using audit cookbook 7.5+ in order to make use of all of the available project rule attributes. Older reports will only have environment and role available as attributes.
```

Comment:
@vjeffrey do we have a reference to this requirement in another section of the docs? if so @blakestier @bcmdarroch could we link to that here as well?